compiler-rt/lib/rtsan/rtsan_interceptors.cpp

```diff
@@ -461,6 +461,19 @@ INTERCEPTOR(void *, valloc, SIZE_T size) {
   return REAL(valloc)(size);
 }
 
+// aligned_alloc was introduced in OSX 10.15
+// Linking will fail when using an older SDK
+#if SANITIZER_APPLE && defined(__MAC_10_15)
```

Comment:
Can we just put this into  sanitizer_platform_interceptors.h ?

Reply:
I moved it up in the latest commit. I initially did not put it on the top level to avoid any possible breakages with other sanitizers.

After looking, it seems like the only sanitizer that interacts with aligned_alloc outside of a *_linux.cpp file is lsan, and I feel reasonably confident this should work for that sanitizer with the weak import defined.